From dc46eb45cb4c6877aaa17cfcdd4f93165e584f08 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Fri, 14 Apr 2023 12:30:38 +0200
Subject: [PATCH 506/529] ARM: configs: add CONFIG_OF_OVERLAY in multiv7 addons
 fragment

On v6.1.12, CONFIG_OF_OVERLAY is no more selected by another config.
So enable it to have dt symbols and overlay available.

Change-Id: Ic8f218ed0b4e408868fc7fd73919c30cb92dc16f
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/300784
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -48,6 +48,7 @@ CONFIG_MTD_SPI_NAND=y
 CONFIG_NAMESPACES=y
 CONFIG_NR_CPUS=2
 CONFIG_NVMEM_STM32_ROMEM=y
+CONFIG_OF_OVERLAY=y
 CONFIG_OPTEE=y
 CONFIG_OVERLAY_FS=y
 CONFIG_PINCTRL_MCP23S08=y
